resident-app-default.properties

```diff
@@ -125,8 +125,7 @@ mosip.kernel.tokenid.length=36
 logging.level.root=WARN
 logging.level.io.mosip=INFO
 # logging.level.io.mosip.kernel.auth.defaultadapter.filter=INFO
-logging.level.io.mosip.kernel.auth.defaultadapter=INFO
-logging.level.org.springframework.http.client=INFO
+logging.level.io.mosip.kernel.auth.defaultadapter=DEBUG
```

Comment:
Changes not required as it's used for testing.

Reply:
Done